Branch-SDK/src/main/java/io/branch/referral/DeviceInfo.java

```diff
@@ -91,6 +91,21 @@ void updateRequestWithV1Params(JSONObject requestObj) {
 
             requestObj.put(Defines.Jsonkey.OSVersion.getKey(), SystemObserver.getOSVersion());
 
+            String[] applicableRoutes = new String[]{"install", "open", "event"};
```

Comment:
Having these as bare strings doesn't seem right.   There are other similar strings in ServerRequestInitSession.